### DISCUSSION TITLE
error with netexec

### DISCUSSION THREAD

**Discussion by alienkeric** [Pennyw0rth/NetExec#453] [General]
I have been getting error everytime i want to use nxc, i don't know why but few days ago it was working very well 

Traceback (most recent call last):
  File "/home/cracker/.local/bin/nxc", line 5, in <module>
    from nxc.netexec import main
ModuleNotFoundError: No module named 'nxc'

**Comment by alienkeric** [comment]
i will appreciate for a help, coz i have been working on thir error now for 2 days

**Comment by NeffIsBack** [comment]
Try `pipx reinstall netexec`, that should do the trick

**Reply by alienkeric** [reply]
thanks for the help this worked, can u tell me what was the pb or was there a package that was broken maybe

**Comment by NeffIsBack** [comment]
Hard to say without sitting in front of the machine, but somehow pipx didn't find the dependency "nxc" anymore. That could be a user error (moved the folder or something), but also a problem with pipx itself (maybe it didn't detect the package anymore because of changes or something). When it reinstalls itself it rebuilds that dependencies and therefor should be present after that

**Reply by alienkeric** [reply]
oky now i get it, thanks for the help :nerd_face:
